.github/workflows/validate.yml

```
@@ -6,6 +6,10 @@ on:
       - main
   pull_request:
   workflow_dispatch:
+  push:
```

actionlint (1.7.7)

key "push" is duplicated in "on" section. previously defined at line:4,col:3

(syntax-check)